docs/pages/platform/upgrading/1.2.mdx

```diff
@@ -5,257 +5,385 @@ meta:
+      
+      const liveblocks = new Liveblocks({
+        secret: "sk_prod_xxxxxxxxxxxxxxxxxxxxxxxx",
+      });
+      
+      export function POST() {
+        const user = { id: "olivier@example.com", info: { name: "Olivier" }};
+  
+        const session = liveblocks.prepareSession(
+          user.id,
+          { userInfo: user.info } // Optional
+        );
+
+        session.allow("my-room-name", session.FULL_ACCESS);
+      }
+      ```
+    </StepContent>
+
+  </Step>
+  
+  <Step>
+    <StepTitle>Authorize the user and return the result</StepTitle>
+    <StepContent>
+    Give if the current user access to the room, with either `session.FULL_ACCESS`, or `session.READ_ACCESS`.
+      ```tsx highlight="17,18"
+      import { Liveblocks } from "@liveblocks/node";
+      
+      const liveblocks = new Liveblocks({
+        secret: "sk_prod_xxxxxxxxxxxxxxxxxxxxxxxx",
+      });
+      
+      export function POST() {
+        const user = { id: "olivier@example.com", info: { name: "Olivier" }};
+  
+        const session = liveblocks.prepareSession(
+          user.id,
+          { userInfo: user.info } // Optional
+        );
+
+        session.allow("my-room-name", session.FULL_ACCESS);
+
+        const { status, body } = await session.authorize();
+        return body;
+      }
+      ```
+    </StepContent>
+
+  </Step>
+  
+  <Step>
+    <StepTitle>Point to the new endpoint</StepTitle>
+    <StepContent>
+    Update your `liveblocks.config.ts` file to connect to the new endpoint.
+    ```ts file="liveblocks.config.ts" highlight="2"
+      const client = createClient({
+        authEndpoint: "/api/liveblocks-auth",
+      });
+      ```
+    </StepContent>
+  </Step>
+  
+  <Step>
+    <StepTitle>You’re migrated!</StepTitle>
+    <StepContent>
+    You’ve successfully migrated, and now have a similar authentication set up as before! However, there are other new features we can take advantage of.
+    </StepContent>
+  </Step>
+  
+  <Step lastStep>
+    <StepTitle>Bonus: Give access to multiple rooms</StepTitle>
+    <StepContent>
+    With Liveblocks 1.2, you can also give access to multiple rooms, or even use a prefix-based wildcard in the room name, enabling any amount of rooms! You can learn more about this in our [access token](/docs/rooms/permissions/access-token#patterns) guide.
+      ```tsx highlight="15-17"
+      import { Liveblocks } from "@liveblocks/node";
+      
+      const liveblocks = new Liveblocks({
+        secret: "sk_prod_xxxxxxxxxxxxxxxxxxxxxxxx",
+      });
+      
+      export function POST() {
+        const user = { id: "olivier@example.com", info: { name: "Olivier" }};
+  
+        const session = liveblocks.prepareSession(
+          user.id,
+          { userInfo: user.info } // Optional
+        );
+
+        session.allow("my-room-name", session.FULL_ACCESS);
+        session.allow("my-other-room", session.READ_ACCESS);
+        session.allow("my-room-*", session.FULL_ACCESS);
```

Comment:
Maybe use read access on the line of the example? Otherwise the first `.allow()` line would be pointless, as it's already covered by this pattern.